decoder/cgroup.go

```diff
@@ -1,15 +1,15 @@
- package decoder
+package decoder
```

Comment:
Could you open a separate PR for this to keep things separate?

Reply:
https://github.com/cloudflare/ebpf_exporter/pull/105

Reply:
There are no changes to the cgroup decoder. It should work just the way it is.